Dockerfile

```diff
@@ -76,6 +76,12 @@ COPY --from=build_base_server "/simpleui-server/dist.tgz" "/tmp/server_dist.tgz"
 # Copy the deployment/run scripts
 COPY "deploy_docker/*.sh" "/scripts/"
 
+# Tini init-system
+ENV TINI_VERSION v0.19.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+
+
```

Comment:
You want to move this to be above the simpleui-specific stuff for caching purposes. Cause docker is gonna have to re-download this every time you change anything in simpleui, which is unnecessary. Move this to line 55

Reply:
Good point, I just pushed some commits that should cover this and and removing the exec command